### PR TITLE
regenerate patches: v3.9.9 --> v3.9.10

### LINKED ISSUE #540: Update to 3.9.10
Instructions to update patches are at https://github.com/conda-forge/python-feedstock/blob/master/recipe/patches/README.md
if anybody wants to give a try.

cc @conda-forge/python, @xhochy

### PR DESCRIPTION
**Pull request by ngam**
Followed instructions #540 

Fixes #540 

cc @isuruf

Of note: patch no. 0008

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ngam** [comment]
@conda-forge-admin, please rerender

**Comment by ngam** [comment]
@conda-forge-admin, please rerender

**Review by xhochy — CHANGES_REQUESTED** [review]
I would like to push the fix for https://github.com/conda-forge/python-feedstock/issues/541 on top here. Doing it first for 3.10 though

**Comment by ngam** [comment]
> I would like to push the fix for #541 on top here. Doing it first for 3.10 though

Take your time --- I am not really a maintainer here or knowledgeable of what's going on, just saw #540 and I thought it would offer to do a mundane thing to help out the general push. Let me know if I can help or if you want me to copy whatever you do for 3.10 :)

File: recipe/patches/0008-bpo-22699-Allow-compiling-on-debian-ubuntu-with-a-di.patch
Comment (xhochy) [inline recipe/patches/0008-bpo-22699-Allow-compiling-on-debian-ubuntu-with-a-di.patch:49]: This should probably be lowercase
```suggestion
             '%s -print-multiarch > %s 2> /dev/null' % (cc, tmpfile))
```

**Comment by xhochy** [comment]
@ngam I will push here when #543 is merged but can you have a look at the one comment I made?

File: recipe/patches/0008-bpo-22699-Allow-compiling-on-debian-ubuntu-with-a-di.patch
Comment (ngam) [reply recipe/patches/0008-bpo-22699-Allow-compiling-on-debian-ubuntu-with-a-di.patch:49]:
```
+        cc = sysconfig.get_config_var('CC')
```

Comment (ngam) [reply recipe/patches/0008-bpo-22699-Allow-compiling-on-debian-ubuntu-with-a-di.patch:49]: I was prompted to fix patch 0008, and I elected to incorporate both from head and commit

Comment (ngam) [reply recipe/patches/0008-bpo-22699-Allow-compiling-on-debian-ubuntu-with-a-di.patch:49]: That's why I noted above to pay attention to patch no. 0008 in case there are issues that aren't clear to me. I think it should be okay, but I someone should review it

**Comment by xhochy** [comment]
Doing this now

**Comment by ngam** [comment]
do you want me to give you access to my fork?

**Comment by xhochy** [comment]
> do you want me to give you access to my fork?

👍 not needed, I have already to this branch as I'm part of `conda-forge/{python,core}`

File: recipe/patches/0008-bpo-22699-Allow-compiling-on-debian-ubuntu-with-a-di.patch
Comment (xhochy) [reply recipe/patches/0008-bpo-22699-Allow-compiling-on-debian-ubuntu-with-a-di.patch]: Applied it as I had to lowercase it also the last time when did update this patch.

**Review by xhochy — APPROVED** [review]

File: recipe/patches/0008-bpo-22699-Allow-compiling-on-debian-ubuntu-with-a-di.patch
Comment (ngam) [reply recipe/patches/0008-bpo-22699-Allow-compiling-on-debian-ubuntu-with-a-di.patch]: Yes, but last time this line `cc = sysconfig.get_config_var('CC')` wasn't there. It's okay, as long as it passes :) 

Thank you for taking care of this!!!

**Comment by ngam** [comment]
Thanks @xhochy for taking care of this!

**Comment by ocefpaf** [comment]
Thanks both of you for the refreshed patches and package size fix!

**Pull request merged**
